mlir/tools/mlir-tblgen/LLVMIRIntrinsicGen.cpp

```diff
@@ -24,44 +24,51 @@
 #include "llvm/TableGen/Record.h"
 #include "llvm/TableGen/TableGenBackend.h"
 
-static llvm::cl::OptionCategory intrinsicGenCat("Intrinsics Generator Options");
+using llvm::Record;
+using llvm::RecordKeeper;
+using llvm::Regex;
+using llvm::SmallVector;
+using llvm::StringRef;
```

Comment:
Could you just add a `using namespace mlir`? That should cover the basic ones.

Reply:
Done